File: angle.js/test/comment.test.js
require('./angle_base_test');

class CommentTest extends ParserBaseTest{

	test_python_comment(){
        result_be(`1 # 3`, 1);
    }
    test_java_comment(){
        this.result_be(`1 // 3`, 1);
    }
    test_java_block_comment(){
        this.result_be(`1 /*3*/ +3`, 4);
    }
    test_bad_comment(){
        this.skip();
        this.result_be(`1\n--no comment`, 1);

    }
}

exports.test_parser=test=>{
	// parse(`1+2=3`)
	test.ok(true, "this assertion should pass");
	test.done();
}

register(CommentTest,module)

File: tests/comment_test.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import angle
from parser_test_helper import *


class CommentTest(ParserBaseTest):

    def setUp(self):
        context.testing = True
    # 	context.use_tree=False

    def test_python_comment(self):
        assert_result_is('1 # 3', 1)

    def test_java_comment(self):
        assert_result_is('1 // 3', 1)

    def test_java_block_comment(self):
        assert_result_is('1 /*3*/ +3', 4)

    def test_bad_comment(self):
        skip()
        assert_result_is('1\n--no comment', 1)
